src/script_chainer/win_exe/script_runner.py

```
@@ -339,6 +341,43 @@ def run_script(script_config: ScriptConfig) -> None:
+    except Exception as e:
+        print_message(f'读取 Python 脚本失败 {display_name}: {e}', level='ERROR')
+        log.error('读取 Python 脚本失败', exc_info=True)
+        return
+
+    if not code or not code.strip():
+        print_message(f'Python 脚本为空 跳过 {display_name}')
+        return
+
+    print_message(f'执行 Python 脚本 {display_name}...')
+    try:
+        exec(compile(code, script_path, 'exec'), {'__builtins__': __builtins__})
```

Ruff (0.15.10)

[error] Use of `exec` detected

(S102)